public/css/screen.styl

```diff
@@ -501,87 +501,336 @@ user-admin-border-size = 2px
 
 }
 
-#session-list {
+#topic-list {
   padding-top: 5px;
 }
 
-/*
- * Session display on event page
- */
+/* 
+  * Topic display on event page 
+*/
 
-.session {
+/* FF 3.5+ */
+body:not(:-moz-handler-blocked) .topic { 
+  width: 98%; 
```

Comment:
I added descriptive comment for overrides, but later took them out as it does not seem we need them. 
https://github.com/drewww/unhangout/commit/171d4a7b64dd60b3c44c3c72079224608f045509
This also fixes the unwanted line-wrap in approved session in Firefox browser.